test/primitiveUtils/primitiveUtils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { DeployHelper } from "../../helpers/DeployHelper";
+import { expect } from "chai";
+
+let utilsWrapper;
+
+before(async () => {
+  const deployHelper = await DeployHelper.initialize();
+  utilsWrapper = await deployHelper.deployPrimitiveTypeUtilsWrapper();
+});
+
+describe("convert address to uint and back", function () {
+  it("convert address to uint and back", async () => {
+    const address = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266";
+    const uint256 = await utilsWrapper.addressToChallenge(address);
+    const convertedAddress = await utilsWrapper.challengeToAddress(uint256);
+    expect(convertedAddress).eq(address);
```

Comment:
Intuitively I feed that although this test is ok but it is better to test each of the functions separately as well.

Reply:
added more tests